inst/scripts/make-metadata.R

```diff
@@ -21,7 +22,8 @@ main.data <- data.frame(
     "Human hematopoietic cell microarray",
     "Human cortex development scRNA-seq",
     "Human pancreatic cell scRNA-seq (inDrop)",
-    "Human pancreatic cell scRNA-seq (SmartSeq2)"
+    "Human pancreatic cell scRNA-seq (SmartSeq2)",
+    "Mouse RNA-seq from 321 cell types"
```

Comment:
get the "atlas" part of the name in there

Reply:
See 5760d0e